recipes/sdformat/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+import os
+
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class SDFormat(ConanFile):
+    name = "sdformat"
+    license = "Apache-2.0"
+    homepage = "http://sdformat.org/"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Simulation Description Format (SDFormat) parser and description files."
+    topics = ("ignition", "robotics", "simulation")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"fPIC": [True, False]}
```

Comment:
Looks like sdformat only supports shared out-of-the box.
See https://github.com/osrf/sdformat/blob/144f82cf8b994abd2a566a7bbe6ae873a9d92403/cmake/SDFUtils.cmake#L56
Have you looked at patching it to support static builds?
Or is there interest upstream to add this possibility?

Reply:
https://github.com/osrf/sdformat/pull/394